hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java

```diff
@@ -503,6 +504,40 @@ public boolean scheduleExists(ScheduleID id) {
 		return Optional.of(info.build());
 	}
 
+	public Optional<List<TokenNftInfo>> infoForAccountNfts(AccountID aid, long start, long end) {
+		var account = accounts().get(fromAccountId(aid));
+		if (account == null) {
+			return Optional.empty();
+		}
+
+		List<TokenNftInfo> nftInfos = new ArrayList<>();
+		var uniqueTokensMap = uniqueTokens.get();
+		uniqueTokenAccountOwnerships.get()
+				.get(fromGrpcAccountId(aid), (int)start, (int)end).forEachRemaining(nftId -> {
```

Comment:
We need to communicate with the platform team about the start/end indexes. Ideally they should be `long`